src/tests/types/message.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Message, MessageType } from "../../types/message";
+
+describe("Message", () => {
+  const rawMsg = `{
+  "type": "Raw",
+  "data": [
+    {
+      "level": "Error",
+      "message": "the tool failed",
+      "time": "2012-04-23T18:25:43.511Z",
+      "extra": {
+        "role": "sdk automation"
+      }
+    }
+  ]
+}`;
+
```

Comment:
How about store JSON as file not in source code?

Reply:
thanks @ruowan  good suggestion, have updated, plz help review again.